code/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
```

Comment:
I think that some [project metadata](https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata) is missing here.

Reply:
Do you think we should duplicate it from `setup.cfg` file?

Reply:
AFAIK `setup.cfg` was an artifact of `setuptools` and is no longer required for configuring a package (but `setuptools` and other tools maybe still use it).

The `pyproject.toml` file was introduced more recently and is supposed to not be specific to any single build tool. 

So, if it's true that `setuptools` works the same without `setup.cfg`, then I'd move the data there to `pyproject.toml`. Otherwise, yes, I'd like to have that duplicated.